### DISCUSSION TITLE
Error: not available with kip-existing subtitles

### DISCUSSION THREAD

**Discussion by Ichicoro-dev** [crunchy-labs/crunchy-cli#371] [Q&A] [answered]
I am using the command 
`crunchy-cli download -skip-existing -a en-US -o "S{season_number}E{episode_number} - {title}.mkv" https://www.crunchyroll.com/series/G6GG38246/spice-and-wolf`

However I get the following:
:: √ Logging in                                                                                                         
:: √ Parsed url 1
:: → Spice and Wolf season 1 is not available with kip-existing subtitles
:: → Spice and Wolf season 2 is not available with kip-existing subtitles
:: √ Skipping url 1 (no matching videos found)

Anyone have a solution? I'm on v3.4.0

**Comment by bytedream** [comment, marked as answer]
You have a missing `-`, it should be `--skip-existing` instead of `-skip-existing`
